pkg/network/master/master.go

```diff
@@ -94,7 +94,7 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, networkClient networkc
 			log.Infof("Created ClusterNetwork %s", common.ClusterNetworkToString(configCN))
 
 			if err = master.checkClusterNetworkAgainstClusterObjects(); err != nil {
-				log.Errorf("WARNING: cluster contains objects incompatible with new ClusterNetwork: %v", err)
+				log.Warningf("Cluster contains objects incompatible with new ClusterNetwork: %v", err)
```

Comment:
I had made this a log.Error intentionally to make it more likely to get noticed in the logs. (You can remove the "WARNING" if you want, but I think this should stay as log.Error.)